### DISCUSSION TITLE
ifm3d version for O3Dxxx and O3X1xx

### DISCUSSION THREAD

**Discussion by coa06yl** [ifm/ifm3d#395] [General]
**It is said only ifm3d 0.20.x is compatible with O3Dxxx and O3X1xx? Does it mean I can't use ifm3d-ros version 0.5.1 with ifm3d version 0.9.2? Thanks!** 
![image](https://user-images.githubusercontent.com/7136290/236398906-a65428cb-4e4a-40be-99a1-e1ef703f0ae2.png)

![Screenshot from 2023-05-05 15-50-18](https://user-images.githubusercontent.com/7136290/236404301-c3908bb6-d5ed-43a8-b14a-7e5029757745.png)

**Comment by lola-masson** [comment]
Hi @coa06yl, 
Versions prior 0.20.x should also be compatible with the O3X. Just make sure you use the right firmaware version, according to this table: https://github.com/ifm/ifm3d/blob/main/doc/sphinx/content/swcompat.md. Other versions might work but are untested. Unfortunately, we have had limited resources to spend on O3X and ROS compatibility.

**Reply by coa06yl** [reply]
Many thanks!

**Comment by lola-masson** [comment]
Hi @coa06yl, 
Versions prior 0.20.x should also be compatible with the O3X. Just make sure you use the right firmaware version, according to this table: https://github.com/ifm/ifm3d/blob/main/doc/sphinx/content/swcompat.md. Other versions might work but are untested. Unfortunately, we have had limited resources to spend on O3X and ROS compatibility.
